micrometer-core/src/test/java/io/micrometer/core/instrument/binder/system/ProcessorMetricsTest.java

```diff
@@ -62,8 +50,7 @@ void hotspotCpuMetrics() {
 
     @Test
     void openJ9CpuMetrics() {
-        OperatingSystemMXBean osBean = ManagementFactory.getOperatingSystemMXBean();
-        assumeTrue(osBean.getClass().getName().equals("com.ibm.lang.management.internal.UnixExtendedOperatingSystem"));
```

Comment:
In addition to access issues, I thought it best to not make our tests depend on the underlying, hidden implementation class. Rather, I thought it would be better to use the public, exported interface to check the JVM implementation. In this way, a future version of the JVM can change and remove implementation classes without us being affected.